.circleci/config.yml

```diff
@@ -112,24 +112,45 @@ jobs:
             ./.circleci/set-nameserver.sh sudo
             sudo apt-get install gettext-base
       - run:
+          name: Cache Prep
           command: |
             sudo mkdir -p /cncf/data
             sudo chown -R circleci:circleci /cncf/
-          name: Cache Prep
       - restore_cache:
           key: cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}
       - run:
+          name: Downloading go deps
           command: |
             ./scripts/prepare-circle-integration-tests.sh
             export COMMIT="${CIRCLE_SHA1:8:8}"
             export CONTAINER_TAG="${COMMIT}"
             # export CONTAINER_FORCE_PULL="true"
             mkdir -p ~/junit/
-            echo "Downloading go deps"
             go mod download
-            echo "Running integration tests"
-            gotestsum --junitfile ~/junit/integration-tests.xml -f standard-verbose ./test/... -timeout 30m -failfast
-            echo "Finish running integration tests"
+          no_output_timeout: 40m
+          environment:
+            KUBECONFIG: /home/circleci/project/data/kubeconfig
+            GO111MODULE: "on"
+      - run:
+          name: Running integration tests - basic
+          command: |
+            gotestsum --junitfile ~/junit/integration-tests-basic.xml -f standard-verbose ./test/... -timeout 30m -failfast -tags=basic
+          no_output_timeout: 40m
+          environment:
+            KUBECONFIG: /home/circleci/project/data/kubeconfig
+            GO111MODULE: "on"
+      - run:
+          name: Running integration tests - recover
+          command: |
+            gotestsum --junitfile ~/junit/integration-tests-recover.xml -f standard-verbose ./test/... -timeout 30m -failfast -tags=recover
```

Comment:
Are we sure it do not overwrite file in second run?

Reply:
I suppose we need to use different files for different parts.

Reply:
Thanks for your feedback! Do you refer to the junit files, as they are indeed different for each test group?